src/controller_examples/rabbitmq_controller/proof/helper_invariants/invariants.rs

```diff
@@ -1112,4 +1096,124 @@ pub proof fn lemma_true_leads_to_always_no_delete_sts_req_is_in_flight(spec: Tem
     );
 }
 
+/// This spec tells that when the reconciler is at AfterGetStatefulSet, and there is a matched response, the reponse must be
```

Comment:
We can probably have a lemma saying that for any get request, the matched get response has the same object key in its content, which applies to all controllers.

Reply:
The thing is we still need a lemma to say that at get stateful set step the pending request is xxx. So I decided to wrap the two. But it does no harm to write the lemma you mentioned.